lnbits/commands.py

```diff
@@ -151,16 +154,41 @@ async def migrate_databases():
 
 
 @db.command("versions")
-def database_versions():
+@coro
+async def db_versions():
     """Show current database versions"""
-    loop = asyncio.get_event_loop()
-    loop.run_until_complete(db_versions())
+    async with core_db.connect() as conn:
+        click.echo(await get_dbversions(conn))
 
 
-async def db_versions():
-    """Show current database versions"""
+@db.command("cleanup-wallets")
+@click.argument("days", type=int, required=False)
+@coro
+async def database_cleanup_wallets(days: Optional[int] = None):
+    """Delete all wallets that never had any transaction"""
+    async with core_db.connect() as conn:
+        delta = days or settings.cleanup_wallets_days
+        delta = delta * 24 * 60 * 60
+        await delete_unused_wallets(delta, conn)
+
+
+@db.command("cleanup-deleted-wallets")
+@coro
+async def database_cleanup_deleted_wallets():
+    """Delete all wallets that has been marked deleted"""
+    async with core_db.connect() as conn:
+        await remove_deleted_wallets(conn)
+
+
+@db.command("cleanup-accounts")
+@click.argument("days", type=int, required=False)
+@coro
+async def database_cleanup_accounts(days: Optional[int] = None):
+    """Delete all accounts that have no wallets"""
     async with core_db.connect() as conn:
-        return await get_dbversions(conn)
+        delta = days or settings.cleanup_wallets_days
+        delta = delta * 24 * 60 * 60
+        await delete_unused_wallets(delta, conn)
```

Comment:
I think this should be `delete_accounts_no_wallets`?

Reply:
yep, copy paste error :o